src/App.js

```diff
@@ -1,26 +1,29 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route, Switch, BrowserRouter } from "react-router-dom";
+import "@patternfly/react-core/dist/styles/base.css";
+import { PageHeader, Page } from "@patternfly/react-core";
+import React from "react";
+import Products from "./Screens/Products"
+import Versions from "./Screens/Versions";
 
-function App() {
+export default function PageLayoutSimpleNav() {
+  const logoProps = {
+    href: "https://github.com/lingostack",
+    target: "_blank",
+  };
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Switch>
+        <>
```

Comment:
We can remove React Fragments from here as we have the <Page /> wrapper.

Reply:
Sure.